internal/oci/container.go

```diff
@@ -183,6 +183,11 @@ func (c *Container) GetStopSignal() string {
 	if c.stopSignal == "" {
 		return defaultStopSignal
 	}
+
+	if _, err := strconv.Atoi(c.stopSignal); err == nil {
+		return c.stopSignal
+	}
```

Comment:
Let's use `unix.SignalName` here to verify the signal number actually exists.

Reply:
The only issue I see with verifying the signal number using Unix.SignalName is that the go unix package has Signal Name defined only upto Signal 31. For few uses cases, we need to use real time signals (SIG 34-64), like to stop systemd pods gracefully SIGRTMIN+3 (signal 37) needs to be sent as stop signal.

Reply:
Oh, okay. Maybe we can extend the range then?

Reply:
@trishnendu wdyt adapting https://github.com/cri-o/cri-o/pull/5366 to fit this case?

Reply:
we could make sure the number is less that RTMAX (defined https://github.com/containers/common/blob/main/pkg/signal/signal_linux.go) and then case to syscall.Signal from there?

Reply:
I wasn't aware of newly added ParseSignal function, #5366 lgtm. 
Right, a upper value check on returned value of ParseSignal is needed